Client.cpp

```diff
@@ -0,0 +1,169 @@
+#include "Client.hpp"
+
+Client::Client()
+{
+	// Initialize Winsock
+	if (WSAStartup(MAKEWORD(2, 2), &_ws) == SOCKET_ERROR)
+	{
+		std::cout << "WSA Failed to Initialize\n";
+		exit(EXIT_FAILURE);
```

Comment:
instead of exit i want you to throw an exception

Reply:
But if you switch to exceptions convert every exit() to an exception hand handle it